backends/sycl-ref/ceed-sycl-ref-operator.sycl.cpp

```diff
@@ -1071,16 +1071,16 @@ static int CeedSingleOperatorAssembleSetup_Sycl(CeedOperator op) {
   CeedCallBackend(CeedGetData(ceed, &sycl_data));
 
   // Kernel setup
-  int elem_per_block   = 1;
-  asmb->elem_per_block = elem_per_block;
-  CeedInt block_size   = elem_size * elem_size * elem_per_block;
+  int elems_per_block   = 1;
+  asmb->elems_per_block = elems_per_block;
+  CeedInt block_size    = elem_size * elem_size * elems_per_block;
 
   /* CeedInt maxThreadsPerBlock = sycl_data->sycl_device.get_info<sycl::info::device::max_work_group_size>();
   bool    fallback           = block_size > maxThreadsPerBlock;
   asmb->fallback             = fallback;
   if (fallback) {
     // Use fallback kernel with 1D threadblock
-    block_size         = elem_size * elem_per_block;
+    block_size         = elem_size * elems_per_block;
     asmb->block_size_x = elem_size;
     asmb->block_size_y = 1;
   } else {  // Use kernel with 2D threadblock
```

Comment:
Since we're here - we should probably ditch the dead code; its in the commit history if we want it back

Reply:
Done in [4826ef4](https://github.com/CEED/libCEED/pull/1302/commits/4826ef44fa2b959953a68071d4331d5eb8192ac4).